Source/driver/_cpp_parameters

```diff
@@ -104,7 +104,12 @@ ppm_predict_gammae           int           0                  y
 ppm_reference_eigenvectors   int           0                  y
 
 # for piecewise linear, reconstruction order to use
-plm_iorder                   int           2                  y
+# 1 = piecewise constant, 2 = MC limiter, >2 = 4th order MC limiter
+plm_iorder                   int           3                  y
```

Comment:
I don't believe we should change the default PLM order, this is a breaking change without a good reason for it.

Reply:
I'm not changing it
I am just changing what the numbers mean.  So 3 is the new 2

Reply:
the idea is 1 is first order (piecewise constant), 2 is second order slopes, >=3 is the 4th order limited slopes

Reply:
If someone has hardcoded plm_iorder in their inputs script, that inputs script will now do something different though, as a result of this change.

Reply:
yes, I know.  I am open to suggestions.  It just seemed odd to me to have 1 = first order, 2 = 4th order, 3 = 2nd order...

Reply:
okay, I've changed how this works.  `plm_iorder` remains the same as before now.  The choice of limiter when using linear slopes is now determined by `plm_limiter`.  I think this is actually a more natural way to do it, since both limiter choices are still linear slopes and therefore second-order.